samples/drivers/ht16k33/README.rst

```diff
@@ -0,0 +1,39 @@
+.. _ht16k33:
+
+HT16K33 LED driver with keyscan
+###############################
+
+Overview
+********
+
+This sample controls the LEDs connected to a `Holtek HT16K33`_ driver,
```

Comment:
Do we need to mention the actual LED hardware that needs to be connected for this sample to work?

Reply:
@dbkinder Thanks for getting back to me. I am not entirely sure I understand what you mean? The actual hardware is a Holtek HT16K33 LED driver (not to confuse with the zephyr software driver for that part).

Reply:
The Holtek HT16K33 LED driver it a hardware device that can control LEDs, but where are the LEDs?

Reply:
@dbkinder Ah, I see what you mean. How about this?